.pre-commit-config.yaml

```
@@ -0,0 +1,20 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 22.3.0 # This should be kept in sync with the version in requirements-dev.in
+    hooks:
+      - id: black
+        language_version: python3
+        files: gnomad_qc
+  - repo: https://github.com/pycqa/isort
```

should we add pydocstyle here too?
```suggestion
  - repo: https://github.com/pycqa/pydocstyle
    rev: 6.0.0 # This should be kept in sync with the version in requirements-dev.in
    hooks:
      - id: pydocstyle
  - repo: https://github.com/pycqa/isort
```

---

Too many errors, I'm trying to get it added now, but in a different PR so the reviewer can actually see how I resolved the errors

---

oh good point, I forgot it would probably edit a bunch of scripts